ci/bamboo/build-manylinux-prototype.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
Mentally subtract the file header, the documentation, the manylinux-specific logic, comments, the ability to make both release and debug builds, and see where you end up: a script without numenta license header, without documentation, without ability to build a manylinux wheel,  fewer comment, no support for both release and debug builds 😄